<!DOCTYPE html>

<html xmlns="http://www.w3.org/1999/xhtml">

<head>

<meta charset="utf-8" />
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<meta name="generator" content="pandoc" />
<meta http-equiv="X-UA-Compatible" content="IE=EDGE" />




<title>CV</title>

<script src="site_libs/jquery-1.11.3/jquery.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/cerulean.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<script src="site_libs/jqueryui-1.11.4/jquery-ui.min.js"></script>
<link href="site_libs/tocify-1.9.1/jquery.tocify.css" rel="stylesheet" />
<script src="site_libs/tocify-1.9.1/jquery.tocify.js"></script>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<link href="site_libs/highlightjs-9.12.0/default.css" rel="stylesheet" />
<script src="site_libs/highlightjs-9.12.0/highlight.js"></script>
<link href="site_libs/font-awesome-5.1.0/css/all.css" rel="stylesheet" />
<link href="site_libs/font-awesome-5.1.0/css/v4-shims.css" rel="stylesheet" />

<style type="text/css">code{white-space: pre;}</style>
<style type="text/css">
  pre:not([class]) {
    background-color: white;
  }
</style>
<script type="text/javascript">
if (window.hljs) {
  hljs.configure({languages: []});
  hljs.initHighlightingOnLoad();
  if (document.readyState && document.readyState === "complete") {
    window.setTimeout(function() { hljs.initHighlighting(); }, 0);
  }
}
</script>



<style type="text/css">
h1 {
  font-size: 34px;
}
h1.title {
  font-size: 38px;
}
h2 {
  font-size: 30px;
}
h3 {
  font-size: 24px;
}
h4 {
  font-size: 18px;
}
h5 {
  font-size: 16px;
}
h6 {
  font-size: 12px;
}
.table th:not([align]) {
  text-align: left;
}
</style>




<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
code {
  color: inherit;
  background-color: rgba(0, 0, 0, 0.04);
}
img {
  max-width:100%;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
summary {
  display: list-item;
}
</style>


<style type="text/css">
/* padding for bootstrap navbar */
body {
  padding-top: 51px;
  padding-bottom: 40px;
}
/* offset scroll position for anchor links (for fixed navbar)  */
.section h1 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h2 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h3 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h4 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h5 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h6 {
  padding-top: 56px;
  margin-top: -56px;
}
.dropdown-submenu {
  position: relative;
}
.dropdown-submenu>.dropdown-menu {
  top: 0;
  left: 100%;
  margin-top: -6px;
  margin-left: -1px;
  border-radius: 0 6px 6px 6px;
}
.dropdown-submenu:hover>.dropdown-menu {
  display: block;
}
.dropdown-submenu>a:after {
  display: block;
  content: " ";
  float: right;
  width: 0;
  height: 0;
  border-color: transparent;
  border-style: solid;
  border-width: 5px 0 5px 5px;
  border-left-color: #cccccc;
  margin-top: 5px;
  margin-right: -10px;
}
.dropdown-submenu:hover>a:after {
  border-left-color: #ffffff;
}
.dropdown-submenu.pull-left {
  float: none;
}
.dropdown-submenu.pull-left>.dropdown-menu {
  left: -100%;
  margin-left: 10px;
  border-radius: 6px 0 6px 6px;
}
</style>

<script>
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark it active
  menuAnchor.parent().addClass('active');

  // if it's got a parent navbar menu mark it active as well
  menuAnchor.closest('li.dropdown').addClass('active');
});
</script>

<!-- tabsets -->

<style type="text/css">
.tabset-dropdown > .nav-tabs {
  display: inline-table;
  max-height: 500px;
  min-height: 44px;
  overflow-y: auto;
  background: white;
  border: 1px solid #ddd;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs > li.active:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li.active:before {
  content: "&#xe258;";
  border: none;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs > li.active {
  display: block;
}

.tabset-dropdown > .nav-tabs > li > a,
.tabset-dropdown > .nav-tabs > li > a:focus,
.tabset-dropdown > .nav-tabs > li > a:hover {
  border: none;
  display: inline-block;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li {
  display: block;
  float: none;
}

.tabset-dropdown > .nav-tabs > li {
  display: none;
}
</style>

<!-- code folding -->



<style type="text/css">

#TOC {
  margin: 25px 0px 20px 0px;
}
@media (max-width: 768px) {
#TOC {
  position: relative;
  width: 100%;
}
}


.toc-content {
  padding-left: 30px;
  padding-right: 40px;
}

div.main-container {
  max-width: 1200px;
}

div.tocify {
  width: 20%;
  max-width: 260px;
  max-height: 85%;
}

@media (min-width: 768px) and (max-width: 991px) {
  div.tocify {
    width: 25%;
  }
}

@media (max-width: 767px) {
  div.tocify {
    width: 100%;
    max-width: none;
  }
}

.tocify ul, .tocify li {
  line-height: 20px;
}

.tocify-subheader .tocify-item {
  font-size: 0.90em;
}

.tocify .list-group-item {
  border-radius: 0px;
}


</style>



</head>

<body>


<div class="container-fluid main-container">


<!-- setup 3col/9col grid for toc_float and main content  -->
<div class="row-fluid">
<div class="col-xs-12 col-sm-4 col-md-3">
<div id="TOC" class="tocify">
</div>
</div>

<div class="toc-content col-xs-12 col-sm-8 col-md-9">




<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">Home</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        
      </ul>
      <ul class="nav navbar-nav navbar-right">
        <li>
  <a href="about.html">Resume</a>
</li>
<li class="dropdown">
  <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-expanded="false">
    Plots
     
    <span class="caret"></span>
  </a>
  <ul class="dropdown-menu" role="menu">
    <li>
      <a href="plotly.html">Plotly</a>
    </li>
    <li>
      <a href="dashboard.html">Dashboard</a>
    </li>
  </ul>
</li>
<li>
  <a href="mailto:nj2208@cumc.columbia.edu">
    <span class="fa fa-envelope fa-lg"></span>
     
  </a>
</li>
<li>
  <a href="http://github.com/jamespng3/">
    <span class="fa fa-github fa-lg"></span>
     
  </a>
</li>
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div class="fluid-row" id="header">



<h1 class="title toc-ignore">CV</h1>

</div>


<div id="education" class="section level3">
<h3>EDUCATION</h3>
<p>
University of Rochester Rochester, NY <span style="float:right">September 2004-May 2008 </span> <br> Bachelor of Science, Biomedical Engineering <br> * Concentration: Cell and Tissue Engineering <br>
</p>
</div>
<div id="laboratory-experience" class="section level3">
<h3>Laboratory Experience</h3>
<p>Columbia University Mailman School of Public Health New York,NY<br> Center for Infection and Immunity <br> * Staff Associate <span style="float:right">June 2016 - Present</span> <br> * Technician B <span style="float:right">Jan 2010 - June 2016</span><br> * Technician A <span style="float:right">April 2009 - Jan 2010</span><br></p>
</div>
<div id="laboratory-skills" class="section level3">
<h3>Laboratory Skills</h3>
<p>Nucleic acid extraction from various tissue types and cDNA synthesis PCR, SDS-PAGE and agarose gels Purification of PCR products and proteins Molecular Cloning In-Situ hybridization/Immunohistochemistry LIPS (Luciferase Immunoprecipitation) Assay Western Blot Northern Blot Southern Blot Transfections of cells with PCR generated constructs Maintenance and oversight of the UGenius gel imager and UV boxes Maintenance and operational oversight of the easyMAG nucleic acid extraction machine Maintenance and operational oversight of the BDMax Laboratory waste management Able to write standard operating procedures Able to use Geneious for genomic sequence analysis and primer design BSL-3 access and Select Agents Program Approval</p>
</div>
<div id="projects" class="section level3">
<h3>PROJECTS</h3>
<p>Acute Encephalitis Syndrome (AES) outbreak in the Uttar Pradesh province of India (2017-present) <em>Process samples on site in Gorakhpur India for molecular and serological tests </em>Analyzed sequencing data to identify pathogenic agents that may be a probable cause of AES *Analyze serological data to determine positive and negative thresholds for probably agents and define samples as positive or negative for said agent Analyze collected data to find links between case outcomes, molecular data and serological data</p>
<p>CII ArboViroPlex Assay FDA EUA (2016-2017) <em>Develop, multiplex and optimization of an RTqPCR assay to accurately detect Zika virus, Dengue virus, West Nile virus and Chikungunya virus </em>Generate RNA constructs for standards <em>Grow and extract cell culture supernatants for limit of detection tests </em>Help write up and submit an FDA EUA *Process FDA packet to validate assay with samples which we did not know what the results should be.</p>
<p>MERS Surveillance of Camels in Riyahd, Saudi Arabia (August 2016)<br />
<em>Extract several hundred samples in two weeks on location in Riyahd, Saudi Arabia </em>Process samples for RTqPCR in duplicates. *Record results and send to supervisor for final analysis</p>
<p>Pathology of Diseased Tilapia with unknown infectious agent (2015-2016) <em>Preparation of samples for unbiased HTS </em>Analysis of HTS data to identify and sequence infectious agent <em>Designing constructs for northern blot assay </em>In-situ hybridization of paraffin embedded slides using probes designed from HTS data</p>
<p>West African Multiplex Assay (2014-2015) <em>Testing multiplex PCR assay mixes on the BD Max machine </em>Testing compatibility of various sample types on the BD Max machine *Program adjustments to optimize signal output</p>
<p>Pathogen burden of Rats in NYC (2013-2014) <em>Extraction of various tissue types and prepping samples for unbiased high-throughput sequencing </em>Analyzing high-throughput data</p>
</div>
<div id="publications" class="section level3">
<h3>PUBLICATIONS</h3>
<p>Nischay Mishra; Shamsudeen F. Fagbo; Abdulaziz N. Alagaili; Adam Nitido; Simon H Williams; James Ng; Bohyun Lee; Abdulkareem Durosinloun; Joel A Garcia; Komal Jain; Vishal Kapoor; Jon H Epstein; Thomas Briese; Ziad A Memish; Kevin J Olival; W Ian Lipkin. A viral metagenomic survey identifies known and novel mammalian viruses in bats from Saudi Arabia. PLOS ONE (in-press) 09/2018.</p>
<p>Thiago Souza, Yasmine Rangel Vieira, Edson Delatorre, Giselle Barbosa-Lima, Raul Leal Faria Luiz, Alexandre Vizzoni, Komal Jain, Milene Mesquita, Nishit Bhuva, Jan F. Gogarten, James Ng, Riddhi Thakkar, Andrea Surrage Calheiros, Ana Paula Teixeira Monteiro, Patricia T Bozza, Fernando A Bozza, Diogo A Tschoeke, Luciana Leomil, Marcos Cesar Lima de Mendonca, Cintia Damasceno dos Santos Rodrigues, Maria C Torres, Ana Maria Bispo de Filippis, Rita Maria Ribeiro Nogueira, Fabiano L. Thompson, Cristina Lemos, Betina Durovni, Jose Cerbino-Neto, Carlos M Morel, Ian Lipkin, Nischay Mishra. Emergence of the East-Central-South-African genotype of Chikungunya virus in Brazil and the city of Rio de Janeiro may have occurred years before surveillance detection. 21 Dec 2018, bioRxiv, doi: <a href="https://doi.org/10.1101/502443" class="uri">https://doi.org/10.1101/502443</a>.</p>
<p>Mishra N, Caciula A, Price A, Thakkar R, Ng J, Chauhan L. V, Jain K, Che X, Espinosa D. A, Montoya Cruz M, Balmaseda A, Sullivan E. H, Patel J. J, Jarman R. G, Rakeman J. L, Egan C. T, Reusken C, Koopmans M, Harris E, Tokarz R, Briese T, Lipkin W. I. (2018). Diagnosis of Zika Virus Infection by Peptide Array and Enzyme-Linked Immunosorbent Assay. mBio, 9(2), e00095-18. <a href="doi:10.1128/mBio.00095-18" class="uri">doi:10.1128/mBio.00095-18</a></p>
<p>Bacharach E, Mishra N, Briese T, Zody MC, Kembou Tsofack JE, Zamostiano R, Berkowitz A, Ng J, Nitido A, Corvelo A, Toussaint NC, Abel Nielsen SC, Hornig M, Del Pozo J, Bloom T, Ferguson H, Eldar A, Lipkin WI. Characterization of a Novel Orthomyxo-like Virus Causing Mass Die-Offs of Tilapia. MBio. 2016 Apr 5;7(2). pii: e00431-16. doi: 10.1128/mBio.00431-16</p>
<p>Firth C, Bhat M, Firth MA, Williams SH, Frye MJ, Simmonds P, Conte JM, Ng J, Garcia J, Bhuva NP, Lee B, Che X, Quan PL, Lipkin WI. Detection of zoonotic pathogens and characterization of novel viruses carried by commensal Rattus norvegicus in New York City. MBio. 2014 Oct 14;5(5):e01933-14. doi: 10.1128/mBio.01933-14</p>
<p>OTHER SKILLS • Able to use Microsoft Office including Microsoft Word, Microsoft Power Point, Microsoft Excel and Microsoft Outlook • Able to use Adobe Photoshop • Able to analyze, prepare and present data to peers • Fluent In Cantonese • Able to input data into LIMS • Basic coding in RStudio</p>
<p>CERTIFICATES • Blood borne pathogen safety • Laboratory Hazardous Waste Safety • Use of Formaldehyde safety • Recombinant DNA training • Biosafety Viral vector training • HIPAA training • Human subjects protection training</p>
<p>REFERENCES References available upon request</p>
</div>



</div>
</div>

</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.header').parent('thead').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- tabsets -->

<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});

$(document).ready(function () {
  $('.tabset-dropdown > .nav-tabs > li').click(function () {
    $(this).parent().toggleClass('nav-tabs-open')
  });
});
</script>

<!-- code folding -->

<script>
$(document).ready(function ()  {

    // move toc-ignore selectors from section div to header
    $('div.section.toc-ignore')
        .removeClass('toc-ignore')
        .children('h1,h2,h3,h4,h5').addClass('toc-ignore');

    // establish options
    var options = {
      selectors: "h1,h2,h3",
      theme: "bootstrap3",
      context: '.toc-content',
      hashGenerator: function (text) {
        return text.replace(/[.\\/?&!#<>]/g, '').replace(/\s/g, '_').toLowerCase();
      },
      ignoreSelector: ".toc-ignore",
      scrollTo: 0
    };
    options.showAndHide = true;
    options.smoothScroll = true;

    // tocify
    var toc = $("#TOC").tocify(options).data("toc-tocify");
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
